tests/python/unittest/test_arith_intset.py

```diff
@@ -370,7 +370,8 @@ def test_union_lower_bound():
     pos_inf = tvm.arith.int_set.pos_inf()
     set_0 = tvm.arith.IntervalSet(min_value=neg_inf, max_value=0)
     set_1 = tvm.arith.IntervalSet(min_value=1, max_value=pos_inf)
-    result = tvm.arith.int_set.union_lower_bound([set_0, set_1])
+    set_2 = tvm.arith.IntervalSet(min_value=pos_inf, max_value=neg_inf)
+    result = tvm.arith.int_set.union_lower_bound([set_0, set_1, set_2])
```

Comment:
it is better to preserve original case and just append new cases

Reply:
code updated，thanks for your great advice!